### DISCUSSION TITLE
[✨] Add ability to customize the "@" symbol for defining the layout name for a route

### DISCUSSION THREAD

**Discussion by Raiondesu** [QwikDev/qwik-evolution#176] [Proposals For Qwik]
### Is your feature request related to a problem?

Some environments, build systems, or code linting configs may reject or misinterpret the `@` used as a layout name marker in the route name. Same can be said of some other directory structure mandates in `qwik-city`.

### Describe the solution you'd like

An ability to customize at least the layout name marker using the qwik-city options object.

Something like this:
```ts
import { defineConfig } from 'vite';
import { qwikVite } from '@builder.io/qwik/optimizer';
import { qwikCity } from '@builder.io/qwik-city/vite';
import tsconfigPaths from 'vite-tsconfig-paths';

export default defineConfig(() => {
  return {
    plugins: [
      qwikCity({
        structure: {
          layoutNameMarker: ':' /// Now it's possible to set layout names with `index:layout-name.tsx`
        }
      }),
      qwikVite(),
      tsconfigPaths()
    ],
  };
});
```

### Describe alternatives you've considered

Maybe it would have been possible to allow the route to provide the `layout` export, like with the request handlers (`onGet`, `onRequest`, etc.), but this would overclutter the route file even more while also removing the feature of knowing the layout for the route at a glance in the file explorer when utilizing this approach.

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
